packages/app-mobile/root.tsx

```diff
@@ -872,7 +872,8 @@ class AppComponent extends React.Component {
 			Config: { screen: ConfigScreen },
 		};
 
-		const statusBarStyle = theme.appearance === 'light' ? 'dark-content' : 'light-content';
+		// const statusBarStyle = theme.appearance === 'light-content';
+		const statusBarStyle = 'light-content';
```

Comment:
I don't really get why we now hardcode the light theme? Before it would change depending on the theme, which seems like the right thing to do?

Reply:
This is because we're now raising the dark area of the light theme & dark themes to behind the status bar, therefore having them still black on a dark background would render them barely visible so setting to light resolves this issue